packages/bp/src/core/app/server.ts

```diff
@@ -451,6 +453,9 @@ export class HTTPServer {
       const totalEnv = `
           (function(window) {
               ${commonEnv}
+              window.APP_NAME = "${branding.title}";
+              window.APP_FAVICON = "${branding.favicon}";
+              window.APP_CUSTOM_CSS = "${branding.customCss}";
```

Comment:
**question:** do you check for undefined values ?

Reply:
@EFF  We don't, but it's because that **.getBrandingConfig** will always return either the default configuration or the one set up by the user. https://github.com/botpress/botpress/blob/0f73cc75f40d25fda9066b12aad94e8f839d668e/packages/bp/src/core/config/config-loader.ts#L284